### ISSUE TITLE
Non-english character support in TTS Generator

### ISSUE THREAD

**Issue by ytt246** [erew123/alltalk_tts#188]
Currently TTS Generator doesn't seem to work with all languages supported by XTTS V2. For example, using arabic, chinese, and japanese characters results in an audio clip of only a second long of incoherent mumbling. Not sure what could've caused this, but it works fine in text-gen-webui

**Comment by erew123** [comment]
Hi @ytt246 

Ive made a small update the the TTS generator you can git pull and see if that changes anything.

Personally I do not speak or read arabic, chinese, and japanese, so if you are able to do so and still have an issue, Perhaps you would send me **short** examples of the text you are trying to generate within the TTS generator then output from the command prompt/terminal so that I can understand what it is or isnt passing over.

Thanks

**Comment by ytt246** [comment]
The problem still exists. I've attached the command prompt output screenshot with three tests:

1. this is a short test using english
2. 这是一个使用中文的测试
3. これは日本語を使ったテストです

![tts](https://github.com/erew123/alltalk_tts/assets/11301086/b621910d-cbd9-4b5a-8412-3a314d24c63b)
[ch-jp-test-audio.zip](https://github.com/erew123/alltalk_tts/files/15119529/ch-jp-test-audio.zip)

It has the same behaviour with Arabic. 

In the case of Korean (이건 한국어로 된 시험이에요), nothing happens after clicking "Generate TTS", no new lines show up and no audio file is generated.

In the case of Hindi (यह हिंदी में एक परीक्षा है), the characters are sent to the command prompt but no audio file is generated.

![hindi](https://github.com/erew123/alltalk_tts/assets/11301086/b9f5a802-d6a9-4864-bc9a-40d1576a8445)

**Comment by erew123** [comment]
Hi @ytt246 

**Hindi**
Hindi is undocumented by Coqui and is only supported on the 2.0.3 model. Please see my notes on this [here](https://github.com/erew123/alltalk_tts/issues/178) regarding setting up the 2,0,3 model on AllTalk 1.9 and also the reference from the main github page.

![image](https://github.com/erew123/alltalk_tts/assets/35898566/620d4b63-a088-4c0b-9bf6-21b49570609d)

With the other languages, I have changed a few bits. If you want to git pull again and give it a go.

![image](https://github.com/erew123/alltalk_tts/assets/35898566/b6b1f856-bca1-401c-9de2-fd213f208d49)

Thanks

**Comment by ytt246** [comment]
I'm glad to report it all works now, thanks!

**Comment by nitinmukesh** [comment]
@ytt246 

Please could you confirm if you checked Hindi was working?

**Comment by ytt246** [comment]
> @ytt246
> 
> Please could you confirm if you checked Hindi was working?

No, I don't speak hindi so I don't know what special characters are used, and can't verify if the results are good or not
